finetuner-workflow/finetune-workflow.yaml

```diff
@@ -91,11 +91,17 @@ spec:
     # CoreWeave region to default to; ORD1 has most of the GPUs.
     - name: region
       value: 'ORD1'
+    # Training GPU - A40, 48gb VRAM
+    - name: trainer_gpu
+      value: 'A40'
+    # Inference GPU - RTX A5000, 24gb VRAM
+    - name: inference_gpu
+      value: 'RTX_A5000'
     # Container images -- generally, don't alter this.
     - name: tokenizer_image
       value: 'ghcr.io/wbrown/gpt_bpe/dataset_tokenizer:ed439c6'
     - name: finetuner_image
-      value: 'docker.io/gooseai/finetuner:rc51'
+      value: 'docker.io/gooseai/finetuner:rc52'
```

Comment:
Should we setup proper CI for this code? I think so

Reply:
Unless we expect everytone to need to do modifications and built their own

Reply:
I completely agree. As a separate task right after delivery of this to the impatiently waiting clients?